web/src/app/workplace/models/task.ts

```diff
@@ -6,7 +6,7 @@ export class Task {
   constructor(public taskId: string,
     public primaryObjRef: ObjectReference = new ObjectReference(),
     public workbasketSummaryResource?: Workbasket,
-    public classificationSummaryResource?: Classification,
+    public classificationSummary?: Classification,
```

Comment:
Is there a reason why you name the variable classificationSummary instead of classification although the type is only classification?

Reply:
The response for a GET of a task returns an object, which has the property classificationSummary, which actually is just a classificationSummary, as i just checked. 
Will update